DOCUMENTATION.md

```diff
@@ -234,6 +264,7 @@ If the request is not successful, the client returns a `Notifications::Client::R
 |`429`|`[{`<br>`"error": "RateLimitError",`<br>`"message": "Exceeded rate limit for key type TEAM/TEST/LIVE of 3000 requests per 60 seconds"`<br>`}]`|Refer to [API rate limits](/ruby.html#api-rate-limits) for more information|
 |`429`|`[{`<br>`"error": "TooManyRequestsError",`<br>`"message": "Exceeded send limits (LIMIT NUMBER) for today"`<br>`}]`|Refer to [service limits](/ruby.html#service-limits) for the limit number|
 |`500`|`[{`<br>`"error": "Exception",`<br>`"message": "Internal server error"`<br>`}]`|Notify was unable to process the request, resend your notification|
+|-|`[{`<br>`"error": "ArgumentError",`<br>`"message": "File size is over the 2 Mb limit")`<br>`}]`|File size was too large, upload a smaller file|
```

Comment:
There are other errors in the python docs related to document uploads (like "unsupported document type" and "document failed virus scan") that should probably be added to this list

Reply:
I've added in the two errors that were missing